content/foundations/color.mdx

```diff
@@ -2,65 +2,191 @@
+Not all colors pair well with each other. There are combinations of backgrounds and foregrounds that guarantee compliance with WCAG contrast guidelines and a wide range of hierarchical relationships between elements. 
+
+#### Pairing color roles
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <Box>
+    <img
+        width="464"
+        alt="Correct colored text and background combinations"
+        src="https://user-images.githubusercontent.com/6951037/141516234-0e267cba-a417-408c-958b-48fcfd1e76b6.png"
+    />
+  <Caption>Don’t combine color text with bg.subtle or bg.emphasis. Colored foregrounds should be combined with their matching backgrounds or bg.default.</Caption>
+    </Box>
+    <Box>   
+        <img
+            width="464"
+            alt="Incorrect colored text and background combinations"
+            src="https://user-images.githubusercontent.com/6951037/143043985-2fb49bb6-9d69-4cd8-9e26-ef5c81147dda.png"
+        />
+        <Caption>Use a link in a footer to provide access to a related page.</Caption>
+    </Box>
+</Grid>
+
+#### Color roles and foregrounds
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <Box>
+    <img
+        width="464"
+        alt="Correct combinations for muted foregrounds"
+        src="https://user-images.githubusercontent.com/6951037/141516384-b6a8625b-29bd-4c5c-be2a-272eda6458c8.png"
+    />
+  <Caption>Only use fg.muted on default, subtle, inset.</Caption>
+    </Box>
+    <Box>   
+        <img
+            width="464"
+            alt="Incorrect combinations for muted foregrounds"
+            src="https://user-images.githubusercontent.com/6951037/141516445-32d33c77-7e79-4b9a-8f4b-dd012158d5a9.png"
+        />
+        <Caption>Never pair emphasis foregrounds with emphasis backgrounds</Caption>
+    </Box>
+</Grid>
+
+#### Color roles recipes
+
+<img width="960" alt="matching color roles variables option 1" src="https://user-images.githubusercontent.com/6951037/141517378-1719360d-e924-46bc-b6e5-4fabcd75a050.png" />
+<img width="960" alt="matching color roles variables option 2" src="https://user-images.githubusercontent.com/6951037/141517387-660b2fef-469b-453b-a112-c8d744103f73.png" />
+<img width="960" alt="matching color roles variables option 3" src="https://user-images.githubusercontent.com/6951037/141517412-08590dc5-bd8d-48de-b130-c16eccf33872.png" />
+<img width="960" alt="matching color roles variables option 4" src="https://user-images.githubusercontent.com/6951037/141517656-e75941ac-6c63-418a-95ea-d687c4d501e7.png" />
+
+## Color outside the functional system
+
+Primer does not provide a presentational system or a data visualization one _yet_. If you or your team at GitHub are working on a feature that have color needs beyond the functional system, please open an issue in the [github/primer](https://github.com/github/primer/issues) repository (Only availanle to GitHub Staff).
 
 ## How to use Primer colors?
```

Comment:
I merged https://github.com/primer/design/pull/199 and fixed the conflict in this PR. In case we still wanna keep that section, we might wanna update its section title since it's now similar to [`How to use color for product UI`](https://github.com/primer/design/pull/192/files#diff-68de77b02787c6f3ce1e2201d79c577c5fa60d2e5bd3a3179f44590149d5d02aR18). Maybe 

```suggestion
## How to use colors in Primer libraries?
```